[CRITICAL] InputMedia was broken

## Context

CPython: 3.9.5 (default, May 24 2021, 12:50:35) [GCC 11.1.0]
OS: Linux-5.12.12-arch1-1-x86_64-with-glibc2.33
aiogram: 2.14.2
aiohttp: 3.7.4.post0
JSON mode: json

InputMedia entities was broken and can't be instantiated.
Affected versions: `>=2.14`

Code:
```python
from aiogram import types

media = types.InputMediaDocument("test")
print(media)
```

Exception:
```python
Traceback (most recent call last):
  File "/tmp/exp/main.py", line 3, in <module>
    media = types.InputMediaDocument("test")
  File "/tmp/exp/venv/lib/python3.9/site-packages/aiogram/types/input_media.py", line 147, in __init__
    super().__init__(
  File "/tmp/exp/venv/lib/python3.9/site-packages/aiogram/types/input_media.py", line 49, in __init__
    super(InputMedia, self).__init__(*args, **kwargs)
  File "/tmp/exp/venv/lib/python3.9/site-packages/aiogram/types/base.py", line 95, in __init__
    self.props[key].set_value(self, value, parent=self)
  File "/tmp/exp/venv/lib/python3.9/site-packages/aiogram/types/fields.py", line 57, in set_value
    value = self.deserialize(value, parent)
  File "/tmp/exp/venv/lib/python3.9/site-packages/aiogram/types/fields.py", line 141, in deserialize
    for item in value:
TypeError: 'NoneType' object is not iterable
```

In aiogram `2.13` it's working fine:
```python
{"media": "test", "type": "document"}
```

I believe the test suite is returning the same error:

```
==================================== ERRORS ====================================
_______________ ERROR collecting tests/types/test_input_media.py _______________
tests/types/test_input_media.py:9: in <module>
    input_media_audio = types.InputMediaAudio(
aiogram/types/input_media.py:178: in __init__
    super().__init__(
aiogram/types/input_media.py:49: in __init__
    super(InputMedia, self).__init__(*args, **kwargs)
aiogram/types/base.py:95: in __init__
    self.props[key].set_value(self, value, parent=self)
aiogram/types/fields.py:57: in set_value
    value = self.deserialize(value, parent)
aiogram/types/fields.py:141: in deserialize
    for item in value:
E   TypeError: 'NoneType' object is not iterable
=========================== short test summary info ============================
ERROR tests/types/test_input_media.py - TypeError: 'NoneType' object is not i...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.61s ===============================
```

--- a/aiogram/types/fields.py
+++ b/aiogram/types/fields.py
@@ -129,6 +129,9 @@ class ListField(Field):
         super(ListField, self).__init__(*args, default=default, **kwargs)
 
     def serialize(self, value):
+        if value is None:
+            return None
+
         result = []
         serialize = super(ListField, self).serialize
         for item in value:
@@ -136,6 +139,9 @@ class ListField(Field):
         return result
 
     def deserialize(self, value, parent=None):
+        if value is None:
+            return None
+
         result = []
         deserialize = super(ListField, self).deserialize
         for item in value:

